README.md

```diff
@@ -62,6 +62,24 @@ aicommits
 aicommits --dry-run
 ```
 
+### Configuration Options
+
+This CLI tool provides the option to configure some of its settings using the `aicommits config` command. The available configuration options are:
+
+-   `OPENAI_KEY`: This option allows you to set the API key for api.openai.com, which is required to access the OpenAI GPT-3 API. You can also set it using an environment variable named OPENAI_KEY, them there's no need to set it using the `aicommits config` command.
+
+-   `generate`: This option determines the amount of commit messages that should be generated by the tool. The more commit messages generated, the higher the cost, as it requires more API calls to the OpenAI GPT-3 API. This can also by done with the cli option `-g`. The default value is 1.
+
+-   `proxy`: This option allows you to set a http/https proxy to use for requests. If you want to clear the proxy option, you can use the command aicommits config set proxy= (note the empty value after the equals sign).
+
+To set a configuration option, use the command `aicommits config set <key=value...>`. For example, to set the API key, you can use `aicommits config set OPENAI_KEY=<your-api-key>`. You can also set multiple configuration options at once by separating them with spaces, like `aicommits config set OPENAI_KEY=<your-api-key> generate=10 proxy=http://your-proxy-url`.
+
+To retrieve a configuration option, use the command `aicommits config get <key...>`. For example, to retrieve the API key, you can use `aicommits config get OPENAI_KEY`. You can also retrieve multiple configuration options at once by separating them with spaces, like `aicommits config get OPENAI_KEY generate`.
+
+Please note that setting any configuration option is optional and the CLI tool will still work without them.
+
+ > Fun fact: This section of the doc was written almost exclusively by ChatGPT.
```

Comment:
🔥🔥🔥

Reply:
Would you actually mind opening a separate PR for this? I can merge it pretty much immediately.

Reply:
https://github.com/Nutlope/aicommits/pull/155 
Okay. I will wait for the merge of that PR and then rebase on that.